cmd/troubleshoot/cli/run_nocrd.go

```diff
@@ -31,6 +34,9 @@ import (
 )
 
 func runTroubleshootNoCRD(v *viper.Viper, arg string) error {
+	fmt.Printf("%s", cursor.Hide())
```

Comment:
Let's put this in the preflight command too?

Reply:
added